tests/surreal_db_repository_test.py

```diff
@@ -0,0 +1,64 @@
+import json
+import unittest
+from unittest.mock import MagicMock, patch
+
+from rococo.models.versioned_model import VersionedModel
+from rococo.repositories.surreal_db_repository import SurrealDbRepository
```

Comment:
@caeser1996 Same as above

Reply:
@caeser1996 I think you missed this line 😄

Reply:
Please check now